ci/platforms/config.gaeac6

```diff
@@ -1,7 +1,18 @@
 #!/usr/bin/bash
 
-export GFS_CI_ROOT=/gpfs/f6/drsa-precip3/scratch/${USER}/GFS_CI_ROOT
+export GFS_CI_ROOT=/ncrc/proj/nggps_emc/${USER}/GFS_CI_CD
```

Comment:
By default, I cannot read the logs from this directory.  I wonder if it is possible to change the permissions so users can view their logs from failing CI tests.  I know these are usually (almost always) posted to GitHub, but there have been some exceptions in the past and it would be good to be able to read these on Gaea.